<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>778</width>
         <height>564</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Fulfilled criteria (required for passing an application):</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="cBox_bug">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="tristate">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'monospace'; font-size:18pt;&quot;&gt;Bug tracker&lt;/span&gt;&lt;span style=&quot; font-family:'monospace';&quot;&gt;&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'monospace';&quot;&gt;http://bugs.maemo.org&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="cBox_lic">
          <property name="text">
           <string>Contains only properly licensed content</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cBox_dub">
          <property name="text">
           <string>No dubious content (adult, violence, racism, etc)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cBox_brk">
          <property name="text">
           <string>No broken basic functionality</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cBox_opt">
          <property name="text">
           <string>Optified</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cBox_cpu">
          <property name="text">
           <string>No unreasonable CPU or resource usage</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cBox_pwr">
          <property name="text">
           <string>No power management issues</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pButton_page">
            <property name="text">
             <string>Use classic (web) interface</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pButton_detail">
            <property name="text">
             <string>QA docs</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Detailed description of issues found (required for failing an application):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="showCommentsButton">
          <property name="text">
           <string>Show comments</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="passButton">
            <property name="text">
             <string>PASS</string>
            </property>
            <property name="icon">
             <iconset resource="kisstester.qrc">
              <normaloff>:/icons/favourite.png</normaloff>:/icons/favourite.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="commentButton">
            <property name="text">
             <string>Comment only</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="failButton">
            <property name="text">
             <string>FAIL</string>
            </property>
            <property name="icon">
             <iconset resource="kisstester.qrc">
              <normaloff>:/icons/buried.png</normaloff>:/icons/buried.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="kisstester.qrc"/>
 </resources>
 <connections/>
</ui>
